data/migrations/20200625220949_create-profile.js

```diff
@@ -3,9 +3,12 @@ exports.up = (knex) => {
     .raw('CREATE EXTENSION IF NOT EXISTS "uuid-ossp"')
     .createTable('profiles', function (table) {
       table.string('id').notNullable().unique().primary();
-      table.string('email');
-      table.string('name');
+      table.string('first_name');
+      table.string('last_name');
+      table.string('email').unique();
+      table.string('location');
       table.string('avatarUrl');
```

Comment:
This should be either 'avatar' or 'avatar_url' - need to keep the same capitalization.